tests/p2p_tests/sync/message_storm.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
```

Comment:
Can you just make the --eosd "--log-level-net-plugin debug" as an argument to test.sh so we don't have two almost identical scripts?

Reply:
that's not the only change. The pnodes, topo, and delay values are also modified, and the setting of debug level output is not optional for this test.  I thought about parameters, but it seemed easier and more sure to just make a new test, Perhaps we can parameterize the test.sh more then make message_storm.sh call that.

Reply:
pnodes, total_nodes, delay, topo are all already parameters.

Reply:
How about you have Ciju combine these tomorrow.